docs/dvc.md

```diff
@@ -0,0 +1,62 @@
+
+## Enabling DVC
+
+Type [`whisk dvc setup`](cli_reference.html#whisk-dvc-setup) to enable DVC within your project. After setup completes, you need a place to store your data. whisk includes a set of commands to automate the setup of an S3 [DVC remote](https://dvc.org/doc/command-reference/remote). To create a remote:
+
+```
+whisk s3 create
+whisk dvc remote-add
+```
+
+This creates an Amazon S3 bucket named `whisk-<project_name>` and creates a DVC remote that uses the S3 bucket.
+
+## Using DVC
+
+The DVC website has [excellent documentation](https://dvc.org/doc) on using DVC. Please refer to their website.
+
+## Example Project
+
+The [Real or Not? NLP with Disaster Tweets](https://github.com/whisk-ml/disaster_tweets) whisk project uses DVC to version control the data download and training stages. This keeps the large dataset and model artifacts outside of version control. As the training stage takes ~20 minutes on a laptop, this can save a significant amount of time when bootstrapping the project. New users can immediately begin using the model after running `dvc pull` versus waiting for the training to complete.
+
+## Heroku deployment
+
+If your app stores artifacts in DVC, those files will need be available by default when the app is [deployed to Heroku](heroku.html). To ensure the artifacts are available, we suggest adding a `.profile` file to the root of your project to pull these artifacts.
+
+[Real or Not? NLP with Disaster Tweets](https://github.com/whisk-ml/disaster_tweets/blob/master/.profile) uses the following `.profile` to do this:
+
+    # .profile
+
+    git init # Needed as dvc pull will not work if this isn't a Git repo.
+    dvc pull train.dvc # Grab only the training output, not all files (don't need raw data)
+
+Heroku [runs the .profile](https://devcenter.heroku.com/articles/dynos#the-profile-file) file before executing the dyno’s command. This ensure that every dyno type has access to the needed model artifacts.
+
+## Python Package
+
+As long as you build the package locally or on a CI server with the DVC files in place, the distribution archive will include any DVC artifacts within `src/<project_name>/artifacts`. However, to support pip installs via a Git repo url (`pip install git+https://url-to-your-git-repo`), you need to get the DVC files as part of the package setup process.
+
+The [Real or Not? NLP with Disaster Tweets](https://github.com/whisk-ml/disaster_tweets/blob/master/setup.py) `setup.py` file shows how to perform [dvc get](https://dvc.org/doc/command-reference/get) as part of the package setup.
+
+## Public Read-only DVC Remote
+
+If you share a public project on GitHub that contains files in a DVC remote, other developers will not be able to access those files by default. To enable read-only access to the DVC remote, run the following commands within your whisk project:
+
+    whisk s3 make-public
+    dvc remote add --local s3 s3://whisk-<project_name>/
```

Comment:
@itsderek23 I haven't dug into the code here yet, but would this work as well here?

```
whisk s3 create
whisk s3 make-public
whisk dvc remote-add
```

Reply:
No. The command order is a workaround to make this work. `whisk dvc remote-add` would add an S3 remote for everyone, but the public does not have access to the S3 bucket.

`dvc remote add --local` ensures that S3 is only available to the project owner. Everyone else gets https.

Reply:
Ah okay sounds good! Thanks for clarifying